content/_redirects

```diff
@@ -54,6 +54,9 @@
 # Replaces Pipelines DSL overview with SDK overview
 /docs/pipelines/sdk/dsl-overview/              /docs/pipelines/sdk/sdk-overview/
 
+# Created a new section for pipelines installation.
+/docs/pipelines/standalone-deployment-gcp/    /docs/pipelines/installation/standalone-deployment/
+
```

Comment:
I already see a section "https://www.kubeflow.org/docs/pipelines/standalone-deployment-gcp/" on kubeflow website, https://www.kubeflow.org/docs/pipelines/standalone-deployment-gcp/

Reply:
This PR moves the existing doc into a new section, and this code redirects incoming links from the existing doc to the new section. 
To test the redirect, try going to this link in the preview of this PR:
https://deploy-preview-1472--competent-brattain-de2d6d.netlify.com/docs/pipelines/standalone-deployment-gcp/